### PR TITLE
Added dependency installer for Arch Linux & some docs

### PR DESCRIPTION
**Pull request by xieve**
The AUR package for the mysql-C++-connector is broken, but there's a fix in the comments and the maintainer seems to be fairly active.

### CODE REVIEW

**Comment by cfr34k** [comment]
Hi xieve,

I finally managed to take a look at your pull request. Unfortunately I can't test it because I don't have an Arch system around, but the list of packages mostly seems ok.
Just one question: why do you have `gunicorn` in the dependencies?

**Comment by xieve** [comment]
That's what I meant by "python dependency confusion". I'm gonna go ahead
and close that PR, then open up another one without all those and with the
few patches.

Am Mo., 3. Juni 2019 um 23:26 Uhr schrieb cfr34k <notifications@github.com>:

**Comment by cfr34k** [comment]
Ok, then I'm ignoring this PR for now and will wait for updates ;)

**Comment by xieve** [comment]
There we go. This should do the job, and I did some documentation, so now there's at least a rough orientation. I may be trying out using traefik as the reverse proxy next week, and generally try to get a more production-ready setup along with some pentesting.

File: dev_run.sh [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 
+#sudo gameserver/docker4bots/00_setup_shm_for_test.sh
```
Comment (cfr34k) [inline dev_run.sh:3]: `sudo` is not needed here, as `00_setup_shm_for_test.sh` uses it where necessary. That script might even produce wrong permissions if it runs as root, so please remove the `sudo` here and in `README.md`.

**Comment by cfr34k** [comment]
Thanks for the update (and especially the README)!

Please note that we already have a reverse proxy script for nginx (PR #3 ), so if you want to try traefik, you might want to base the experiment on the updated `master` branch ;-) .

File: dev_run.sh (hunk 1 above)
Comment (cfr34k) [reply dev_run.sh:3]: sorry, I didn't see the PR ( https://github.com/schlangenprogrammiernacht/gameserver/pull/2 ) in the gameserver repo. Let's discuss this issue there.

**Comment by cfr34k** [comment]
@xieve I merged this as I think the README update is rather important, but reverted the part about `sudo` and only forwarded the gameserver submodule to the current master.

**Comment by xieve** [comment]
Yeah I figured that the sudo part was indeed neccesary. That makes the
dependency script incomplete though, `sudo` is missing. I looked into the
traefik options and I don't think I'm gonna compose a docker image for spn,
so traefik isn't actually a great option. Also, it's meant for integrating
multiple microservices, while spn only works in whole, so that's how it
should be provided, too. But all-in-all, containerizing spn would be both
useful (complicated setup which affects many system parts) and also
interesting (we'd either have to use nested containers or provide the right
interfaces so that the bot containers can run on the host machine directly).

**Pull request merged**